content/How to change the default datepicker on the Django admin page - Jorge Michelena.md

```diff
@@ -0,0 +1,146 @@
+When you register a model in your Django admin page it automatically matches each of the model's fields to
+a widget so you can input data and manage instances of the models easily from the Django admin panel, 
+most of the time the default widgets are enough but in some cases they may not be the most suitable for our particular needs.
+
+For instance, if one of your models has a `DateField` which represents the birth date of a user and you know that for whatever
+reason you will need to create user instances frequently from the admin panelthen the default datepicker may be very inconvenient
+if you want to input realistic data since you can't input the year in a straightforward way and would have to navigate month to month to reach a desired year.
+
+In such a case you may want to change the widget to pick the date but after doing some research may not find a clear explanation
+on how to do this, but you are on luck because not only is it pretty easy but it is also what we will be guiding through in this article! Let's begin.
+
+### Set up
+
+I'm going to asume that you have python and django installed as well as a project ready on which you are going to make these changes.
+In my case I made a very simple project called `project` with just one app called `customers`, and the following layout:
+
+
+```
+| project
+├── customers
+│   ├── __init__.py
+│   ├── admin.py
+│   ├── apps.py
+│   ├── migrations
+│   │   ├── 0001_initial.py
+│   │   ├── __init__.py
+│   ├── models.py
+│   ├── tests.py
+│   └── views.py
+├── db.sqlite3
+├── manage.py
+└── project
+    ├── __init__.py
+    ├── asgi.py
+    ├── settings.py
+    ├── urls.py
+    ├── widgets.py
+    └── wsgi.py
+```
+
+We will be working in the `Customer` model in `customers.py` which looks something like this:
+
+```python
+from django.db import models
+
+class Customer(models.Model):
+    name = models.CharField(max_length=100)
+    email = models.CharField(max_length=100)
+    birth_date = models.DateField()
+
+```
+
+Having registered the model in the admin page then when trying to create/modify a new instance from the admin panel it will look like this
```

Comment:
```suggestion
Having registered the model on the admin page then when we try to create/modify a new instance from the admin panel it will look like this
```